helm/prometheus-agent/values.schema.json

```diff
@@ -206,6 +206,40 @@
                 "retention": {
                     "type": "string"
                 },
+                "scrapeClasses": {
```

Comment:
How about we add a defaultscrapeclass for ours and a scrapeclass value for customers?

Reply:
IIUC, if there is one default scrapeClass defined, all scraping targets that don't specify scrapeClass will inherited of the default one.
So if the customer doesn't update their targets with the specific scrapeClass, it will have our.

Reply:
I meant, have giantswarm defined scrape classes and another field for customers because I do not really like that they can override ours